perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -1309,6 +1309,87 @@ def GetOsInfo(self):
     stdout, _ = self.RemoteCommand('swupd info | grep Installed')
     return "Clear Linux build: {0}".format(regex_util.ExtractGroup(CLEAR_BUILD_REGEXP, stdout))
 
+  def SetupProxy(self):
+    """Sets up proxy configuration variables for the cloud environment."""
+    super(ClearMixin, self).SetupProxy()
+    profile_file = '/etc/profile'
+    commands = []
+    command_template = "grep -qxF '{0}' {1} || echo '{0}' | sudo tee -a {1}"
+
+    if FLAGS.http_proxy:
+      commands.append(command_template.format(
+          'export http_proxy=' + FLAGS.http_proxy, profile_file))
+      commands.append(command_template.format(
+          'export HTTP_PROXY=' + FLAGS.http_proxy, profile_file))
+
+    if FLAGS.https_proxy:
+      commands.append(command_template.format(
+          'export https_proxy=' + FLAGS.https_proxy, profile_file))
+      commands.append(command_template.format(
+          'export HTTPS_PROXY=' + FLAGS.https_proxy, profile_file))
+
+    if FLAGS.ftp_proxy:
+      commands.append(command_template.format(
+          'export ftp_proxy=' + FLAGS.ftp_proxy, profile_file))
+      commands.append(command_template.format(
+          'export FTP_PROXY=' + FLAGS.ftp_proxy, profile_file))
+
+    if FLAGS.no_proxy:
+      commands.append(command_template.format(
+          'export no_proxy=' + FLAGS.no_proxy, profile_file))
+      commands.append(command_template.format(
+          'export NO_PROXY=' + FLAGS.no_proxy, profile_file))
+
+    if commands:
+      self.RemoteCommand(';'.join(commands))
+
+  def ProxyCleanup(self):
```

Comment:
Where is this called?

Reply:
Where is it defined?

https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/search?q=ProxyCleanup&type=Code
We couldn’t find any code matching 'ProxyCleanup' in GoogleCloudPlatform/PerfKitBenchmarker
You could try an advanced search.

Reply:
sorry my apology, ProxyCleanup function only exists in Internal repository, not in upstream. I have removed it.